java/src/org/openqa/selenium/edge/EdgeDriverService.java

```diff
@@ -248,16 +231,24 @@ public Builder withLoglevel(ChromiumDriverLogLevel logLevel) {
      * @return A self reference.
      */
     public Builder withSilent(boolean silent) {
-      if (silent) {
-        this.logLevel = ChromiumDriverLogLevel.OFF;
-      }
       this.silent = false;
       return this;
     }
 
     /**
-     * Configures the comma-separated list of remote IPv4 addresses which are allowed to connect
-     * to the driver server.
+     * Configures the driver server verbosity.
+     *
+     * @param verbose whether verbose output is used
+     * @return A self reference.
+     */
+    public Builder withVerbose(boolean verbose) {
+      this.verbose = false;
```

Comment:
I think you need to assign the value to `this.verbose`, otherwise it gets ignored.